micrometer-metrics-examples/README.adoc

```diff
@@ -0,0 +1,116 @@
+= Vert.x Micrometer Metrics examples
+
+This project shows simple examples using the Vert.x Micrometer Metrics module.
```

Comment:
_These examples demonstrate how the Vert.x Micrometer Metrics module_ + add link to the online documentation

Reply:
Hmmm... you mean _demonstrate how **to use**_ maybe?